packages/components/pager/README.md

```diff
@@ -0,0 +1,147 @@
+<h1 align="center">
+    <a href="https://tradeshift.com/">
+      <img alt="Tradeshift" src="https://tradeshift.com/wp-content/themes/Tradeshift/img/brand/logo-black.png"/>
+    </a>
+</h1>
+
+<h1 align="center">Elements - pager</h1>
+
+<p align="center">
+  Part of the reusable Tradeshift UI Components as Web Components.
+    <a href="https://tradeshift.github.io/elements/?path=/story/ts-pager--default">
+      Demo
+    </a>
+</p>
+
+<p align="center">
+    <a href="https://www.npmjs.com/package/@tradeshift/elements.pager">
+      <img alt="NPM Version" src="https://badgen.net/npm/v/@tradeshift/elements.pager" height="20"/>
+    </a>
+    <a href="https://npmcharts.com/compare/@tradeshift/elements.pager?minimal=true">
+		  <img alt="Downloads per month" src="https://badgen.net/npm/dm/@tradeshift/elements.pager" height="20"/>
+		</a>
+		<a href="https://www.npmjs.com/browse/depended/@tradeshift/elements.pager">
+		  <img alt="Dependent packages" src="https://badgen.net/npm/dependents/@tradeshift/elements.pager" height="20"/>
+		</a>
+</p>
+
+## ➤ Properties
+
+| Property     | Attribute    | Type   | Default                                                                                                                                                                    | Description                                                               |
+| ------------ | ------------ | ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| totalPages   | total-pages  | number | -                                                                                                                                                                          | Total number of pages                                                     |
+| activePage   | active-page  | number | 1                                                                                                                                                                          | Currently active page                                                     |
+| perPage      | per-page     | number | 20                                                                                                                                                                         | Determining maximum number of items in the page, should be 10,20,30,40,50 |
+| translations | translations | object | {page: 'Page', of: 'of', items_per_page: 'Items per page', first_page_tooltip: 'You are already on the first page', last_page_tooltip: 'You are already on the last page'} | Add circular background for icon                                          |
```

Comment:
I believe the description does not match the attribute.

Reply:
I forget it every single F*&^ing time :| 😅